tests/script-based-pre/individual_file_output/individual_file_output.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set +e
+
+OUT_DIR=tmp_sample_crate
+
+# Ensure output folder is clean
+rm -rf ${OUT_DIR}
+
+# Move the original source to the output folder since it will be modified
+cp -r sample_crate ${OUT_DIR}
+pushd $OUT_DIR
+
+echo "Run verification..."
+../../../../scripts/cargo-kani -Z unstable-options --output-into-files
```

Comment:
You should just call `cargo kani` here. I think our test infrastructure should ensure path is correct